### ISSUE TITLE
Question about ginput & other things

### ISSUE THREAD

**Issue by Zayoe4** [hrt/BrawlBot#9]
@hrt 
I am updating g_input_offsets and I wanted to know what I should do next.
![image](https://user-images.githubusercontent.com/10657721/96351153-2a35ec00-107f-11eb-93b1-0248de77a5c3.png)
Here you can see that I found five different addresses for ginput. I was thinking I should do a pointer scan and try to find a base address, but I'm not sure which address I should try to uses, they all have different offsets, but the same address accessing them 3C96B0B0.

Another question I wanted to ask, in your offsets.py file, I noticed that you had several other values behind the base address:
`g_input_offsets = ('Adobe AIR.dll', [0x0131550C, 0x1DC, 0x18, 0x8, 0x98, 0x1F8, 0x4CC])`
0x0131550C is the base address, is everything behind it just offsets? 
The reason I'm asking is because whenever I do a pointer scan, I get down to 80 pointers (of which, 60 are adobeair.dll) and I'm not sure which pointer I should use. I assume I should be able to use any of them since when I restart Brawlhalla, they always point back to where they should. Below I've included the 80 pointers I got from local player:
![image](https://user-images.githubusercontent.com/10657721/96351325-32425b80-1080-11eb-947e-934e379d675c.png)
So let's imagine I used the address at the top of the list in the screenshot. `"Adobe AIR.dll"+01315500`
Would I need to edit the file to read: 
local_ptr_offsets = ('Adobe AIR.dll', [0x01315500,` 
If so, would the offsets be added to the right like this:
local_ptr_offsets = ('Adobe AIR.dll', [0x01315500, 0x7A8, 0x52C, 0xFC, 0x568, 0x370])`
Here is a better example of what I mean:
![image](https://user-images.githubusercontent.com/10657721/96351797-800c9300-1083-11eb-85aa-5dc5abd09e15.png)

TL;DR: 
Question 1: I found six ginputs, which should I use?
Question 2a: I have 60 base address, can I use any of them for local_ptr_offsets?
Question 2b: Is the image below correct?
![image](https://user-images.githubusercontent.com/10657721/96351797-800c9300-1083-11eb-85aa-5dc5abd09e15.png)

Thank you for coming to my TedTalk.

**Comment by Sc0peBetter** [comment]
can you add me on Discord? Sc0pe#9249

**Comment by taohabo** [comment]
do you can add me discord Thien1994#5618
@Zayoe4

**Comment by 0xFFD** [comment]
can you add me pal ? Discord : 0xFFD#2677

**Comment by hrt** [comment]
@0xFFD don't expect much since I'm not active at all with games, but my discord is ttap#4547

**Comment by Ornamus** [comment]
@Zayoe4 I'm very curious how you resolved this, as I'm doing this myself at the moment. Can you add me on Discord? Ornamus#6574
